pkg/virt-launcher/notify-client/client.go

```diff
@@ -270,7 +270,7 @@ func eventCallback(c cli.Connection, domain *api.Domain, libvirtEvent libvirtEve
 		if interfaceStatus != nil || osInfo != nil {
 			event := watch.Event{Type: watch.Modified, Object: domain}
 			client.SendDomainEvent(event)
-			events <- event
+			updateEvents(event, domain, events)
```

Comment:
This might can be removed, in case the only delete with timestamp is what we create on line 247
https://github.com/kubevirt/kubevirt/pull/4253/files#diff-7175e08324a35a860b12d6518d396988R247